### ISSUE TITLE
Incorporate into mainline SWIG repository

### ISSUE THREAD

**Issue by wsfulton** [FMXExpress/swig-delphi#1]
Is there any interest in getting this incorporated into the mainline SWIG repository at https://github.com/swig/swig? We are looking at gathering up all the various good quality SWIG target languages that are in various different branches and releasing into the next major version swig-4.0.

**Comment by FMXExpress** [comment]
Yes, the end goal has always been to get the Delphi version merged into the main version. We have 140+ hours into it.

**Comment by lynxnake** [comment]
@wsfulton, see https://github.com/swig/swig/pull/985 - it is the same code but "rebased" on the main SWIG repo

**Comment by wsfulton** [comment]
Great, I'll be taking a look and comment on it there.

**Comment by wsfulton** [comment]
For future reference and clarification as it confused me, this repository (https://github.com/FMXExpress/swig-delphi), is not a fork of the upstream SWIG repository, however, another FMExpress repository (https://github.com/FMXExpress/swig) is a proper Git fork of the upstream SWIG repository. More recent Delphi development seems to be taking place on the later.

**Comment by lynxnake** [comment]
Exactly.
